### PR TITLE
holdingpen: fix all harvests query

### PR DESCRIPTION
**Pull request by Glignos**
*Fixes all harvests query to introduce workflows that dont
have the 'is-update' key.

Signed-off-by: Glignos <glignos93@gmail.com>

## Checklist:

- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [ ] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: inspirehep/modules/workflows/templates/inspire_workflows/index.html [hunk 1]
```diff
@@ -61,7 +61,7 @@ <h3 class="text-center header-text">
 
       <holding-pen-dashboard-item
         template="{{url_for('static', filename='node_modules/inspirehep-search-js/dist/templates/inspirehep-holdingpen-js/dashboard_stat_block.html')}}"
-        filter_string="source=arXiv&workflow_name=HEP&is-update=false"
+        filter_string="source=arXiv&workflow_name=HEP&q=!_extra_data.is-update:true"
```
Comment (michamos) [inline inspirehep/modules/workflows/templates/inspire_workflows/index.html]: `q=!is-update:true` seems to work also.

Comment (Glignos) [reply inspirehep/modules/workflows/templates/inspire_workflows/index.html]: I think not, it seems that this query returns you the whole set of results.

Comment (michamos) [reply inspirehep/modules/workflows/templates/inspire_workflows/index.html]: you're right! I didn't look close enough :+1:

**Comment by Glignos** [comment]
@drjova can you merge? :)

**Comment by michamos** [comment]
it needs to be rebased first

**Pull request merged**